package.json

```diff
@@ -14,7 +14,7 @@
 		"yarn": ">=1.7.0"
 	},
 	"scripts": {
-		"test": "xo && ava"
+		"test": "xo && node script/test-ava.js"
```

Comment:
You can just do this: https://github.com/sindresorhus/is/blob/9d6c91ee58d14406163861cd95f7d2daceeef035/package.json#L21

Reply:
That wouldn't work on Windows though.